src/main/java/nextstep/courses/service/SessionService.java

```diff
@@ -23,10 +23,11 @@ public Session createSession(Session session){
         return sessionRepository.save(session);
     }
 
-    public void registerStudent(NsUser student, Long sessionId, String paymentId){
+    public void registerStudent(NsUser student, Long sessionId){
         Session session = sessionRepository.findById(sessionId);
-        Payment payment = paymentService.payment(paymentId);
+        Payment payment = paymentService.payment(session.getId(), student.getId());
```

Comment:
```suggestion
        Payment payment = paymentService.findPayment(session.getId(), student.getId());
```

find를 붙이는게 더 자연스러운것 같아요 🤔